google/cloud/security/inventory/pipeline_builder.py

```diff
@@ -49,9 +50,53 @@ def __init__(self, cycle_timestamp, config_path, flags,
         self.flags = flags
         self.api_map = api_map
         self.dao_map = dao_map
+        self.initialized_api_map = {}
```

Comment:
Hmmm, the initialized_api_map is generated internally, so should not be passed in as method parameter.  So, I think it's better to keep the method signature as is.